Model/Config/Frontend/EnableProductProtection.php

```diff
@@ -50,16 +51,22 @@ protected function _getElementHtml(
         \Magento\Framework\Data\Form\Element\AbstractElement $element
     ) {
         if (
-            $this->checkIfV1PPEnabledInAnyStores() &&
-            $this->manager->isEnabled('Extend_Warranty')
+            ($this->checkIfV1PPEnabledInAnyStores() &&
+                $this->manager->isEnabled('Extend_Warranty')) ||
+            !$this->checkIfV2ExtendEnabled()
```

Comment:
the comments above the functions in this file read 
```
This will prevent enabling of product protection on the new module, if it's enabled on the old module.
```
which is true but could also include this new case about Extend itself being disabled